### ISSUE TITLE
Remove logo block if no logo is present

### ISSUE THREAD

**Issue by dan-bowerman** [ramp4-pcar4/storylines-editor#210]
A Storylines Product will not load in the editor unless its intro slide has a logo block. The `src` and `altText` can be blank, and nothing will appear. However, when running accessibility reviews, the empty image is detected as missing alternative text.

Two things to address here:
1. Don't render HTML for the logo if the JSON block for the logo block is empty/blank (A Storylines platform issue, new issue will be required there)
2. Allow the editor to load a page if no logo structure is present at all. 

Related #207.

**Comment by RyanCoulsonCA** [comment]
This was fixed with the PR #214
